src/examples/java/io/synadia/flink/examples/JetStreamExample.java

```diff
@@ -50,7 +51,7 @@ public class JetStreamExample {
     // Set the quiet period longer if you are using ack mode. See notes on ACK_MODE below.
     // Try 3000 or 10000 in ack mode.
     // ------------------------------------------------------------------------------------------
-    public static final int QUIET_PERIOD = 3000;
+    public static final int QUIET_PERIOD = 10000;
```

Comment:
Please return all examples to default to AckPolicy.None. This should always be the default. This includes the quiet period back to 3000. Also update to the comment to reflect ACK_POLICY instead of ACK_MODE